recipes/shield/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools, CMake
+import os
+
+class ShieldConan(ConanFile):
+    name = "shield"
+    version = "0.1"
+    topics = ("shield", "utility", "warnings", "suppression")
```

Comment:
```suggestion
    topics = ("utility", "warnings", "suppression")
```
Do not include project's name

Reply:
done